hydra-node/src/Hydra/Snapshot.hs

```diff
@@ -55,32 +59,50 @@ instance IsTx tx => FromJSON (Snapshot tx) where
       <*> (obj .: "snapshotNumber")
       <*> (obj .: "utxo")
       <*> (obj .: "confirmedTransactions")
+      <*> (obj .: "utxoToDecommit")
 
 instance (Arbitrary (TxIdType tx), Arbitrary (UTxOType tx)) => Arbitrary (Snapshot tx) where
   arbitrary = genericArbitrary
 
   -- NOTE: See note on 'Arbitrary (ClientInput tx)'
-  shrink Snapshot{headId, number, utxo, confirmed} =
-    [ Snapshot headId number utxo' confirmed'
+  shrink Snapshot{headId, number, utxo, confirmed, utxoToDecommit} =
+    [ Snapshot headId number utxo' confirmed' utxoToDecommit'
     | utxo' <- shrink utxo
     , confirmed' <- shrink confirmed
+    , utxoToDecommit' <- shrink utxoToDecommit
     ]
 
 -- | Binary representation of snapshot signatures
 -- TODO: document CDDL format, either here or on in 'Hydra.Contract.Head.verifyPartySignature'
 instance forall tx. IsTx tx => SignableRepresentation (Snapshot tx) where
-  getSignableRepresentation Snapshot{number, headId, utxo} =
+  getSignableRepresentation Snapshot{number, headId, utxo, utxoToDecommit} =
     LBS.toStrict $
       serialise (toData $ toBuiltin $ serialiseToRawBytes headId)
         <> serialise (toData $ toBuiltin $ toInteger number) -- CBOR(I(integer))
         <> serialise (toData $ toBuiltin $ hashUTxO @tx utxo) -- CBOR(B(bytestring)
+        -- TODO: is serializing separately fine or we should get the diff
```

Comment:
do not get this question, is it still open?

Reply:
It is about serialization of the decommit utxo and wether we should have it separate or serialize snapshot utxo with already removed decommit utxo.